### PR TITLE
Fixes #427 when there are no search suggestions drop down is hidden

### LINKED ISSUE #427: Drop down open even when there are no suggestions
**Actual Behaviour**
the drop down of search bar is open even when there are no suggestions.
**Expected Behaviour**

Drop Down needs to be hidden when there are no suggestions.
**Screenshots of the issue**

image

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
Fixes issue #427

Changes: Now drop down is hidden when there no suggestions.

Demo Link: https://susper-pr-430.herokuapp.com/

Screenshots for the change: 

image

### CODE REVIEW

File: src/app/auto-complete/auto-complete.component.ts [hunk 1]
```diff
@@ -10,27 +10,42 @@ import * as fromRoot from '../reducers';
...
       if (query) {
-        this.autocompleteservice.getsearchresults(query).subscribe(res => {
-          if (res[0]) {
-            // console.log(res[1]);
-            this.results = res[1];
-            this.results.concat(res[0]);
-            if ( this.results.length > 5) {
-            this.results = this.results.splice (0, 5);
+        this.query = query;
+
```
Comment (harshit98) [inline src/app/auto-complete/auto-complete.component.ts]: what is this `this.query = query` ?

**Comment by Marauderer97** [comment]
image
Why is there no auto complete being displayed, even when there are suggestions available?
image

**Comment by nikhilrayaprolu** [comment]
this problem persists in susper.com too. it isn't caused by this pull. @Marauderer97

**Comment by Marauderer97** [comment]
@nikhilrayaprolu But look:
image
:worried:

**Comment by nikhilrayaprolu** [comment]
@Marauderer97 look at this
image

**Comment by Marauderer97** [comment]
Ok LGTM now! :smile: 
image
Please solve merge conflicts :smiley:

**Comment by nikhilrayaprolu** [comment]
@Marauderer97 done.

**Pull request merged**